src/main/java/org/junit/rules/RuleChain.java

```diff
@@ -37,6 +37,22 @@
  * finished middle rule
  * finished outer rule
  * </pre>
+ * 
+ * {@code RuleChain} cannot be used to define the order of existing rules.
+ * For example in the below snippet the LoggingRule <i>middle</i> would be executed 
+ * outside as well as inside the {@code RuleChain}:
+ * <pre>
+ * &#064;Rule
+ * public LoggingRule middle = new LoggingRule("middle rule");
+ * 
+ * &#064;Rule
+ * public RuleChain chain = RuleChain
+ *                          .outerRule(new LoggingRule("outer rule"))
+ *                          .around(middle)
+ *                          .around(new LoggingRule("inner rule"));
+ * </pre>
+ * <p>
```

Comment:
It's usually a good idea to take a look at the generated Javadoc locally. You can generate it using Maven. Let me know if you need more detailed instructions.